integration-tests/itest/deploy_test.go

```diff
@@ -143,12 +145,48 @@ var _ = Describe("Application deployment", func() {
 			close(done)
 		}, LongTimeout)
 
-		It("should have configured a flink streamlet", func(done Done) {
+		XIt("should have configured a flink streamlet", func(done Done) {
```

Comment:
Ok. so we keep this as pending for now so it does not break stuff, LGTM.

Reply:
Yeah. It produces a random failure. -- we should test it manually from time to time, though.